### PR TITLE
Prac06 feedback

### PR DESCRIPTION
**Pull request by nora-zyy**
@a-rosa
@AdriantoHartono
@JevanChandra

### CODE REVIEW

File: prac_06/car.py
Comment (AdriantoHartono) [inline prac_06/car.py:1]: Good class code, it written nicely👌

File: prac_06/guitar.py
Comment (AdriantoHartono) [inline prac_06/guitar.py:1]: Good class, it written nicely 👍

File: prac_06/guitar_test.py
Comment (AdriantoHartono) [inline prac_06/guitar_test.py:1]: The code already work as expected, but it will be better if you put it into a function maybe 😅

File: prac_06/guitars.py
Comment (AdriantoHartono) [inline prac_06/guitars.py:1]: Nice code, it can show the result as expected 👍

File: prac_06/languages.py
Comment (AdriantoHartono) [inline prac_06/languages.py:1]: Good code, it can show the result as expected 👍

File: prac_06/programming_language.py
Comment (AdriantoHartono) [inline prac_06/programming_language.py:1]: The class has been written nicely 👍

File: prac_06/used_cars.py
Comment (AdriantoHartono) [inline prac_06/used_cars.py:1]: The code can work properly, and it also written nicely 🙏

File: prac_06/car.py
Comment (JevanChandra) [inline prac_06/car.py:1]: Quite long but it works so good job!

File: prac_06/guitar.py
Comment (JevanChandra) [inline prac_06/guitar.py:1]: Short and very easy to understand!

File: prac_06/guitar_test.py
Comment (JevanChandra) [inline prac_06/guitar_test.py:1]: I agree with adrianto. I think you need to add a function

File: prac_06/guitars.py
Comment (JevanChandra) [inline prac_06/guitars.py:1]: Nice code!

File: prac_06/languages.py
Comment (JevanChandra) [inline prac_06/languages.py:1]: Great Work!

File: prac_06/programming_language.py
Comment (JevanChandra) [inline prac_06/programming_language.py:1]: Had trouble with this section hope you don't mind me taking reference

File: prac_06/used_cars.py
Comment (JevanChandra) [inline prac_06/used_cars.py:1]: Great work!

File: prac_06/guitar.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    def get_age(self):
+        this_year = 2023
+        return this_year - self.year
+
+    def is_vintage(self):
+        return self.get_age() >= 50
+
+
+
```
Comment (a-rosa) [inline prac_06/guitar.py:18]: Missing docstring on all function

File: prac_06/guitars.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+from prac_06.guitar import Guitar
+
+def main():
+
+    print("My guitars!")
+    guitars = []
+    guitars.append(Guitar("Gibson L-5 CES", 1992, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.90))
+    guitars.append(Guitar("Fender Stratocaster", 2014, 765.40))
+
```
Comment (a-rosa) [inline prac_06/guitars.py:10]: The guitars.append information should not be there if it is used to test the program

File: prac_06/languages.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+from prac_06.programming_language import ProgrammingLanguage
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+print(python)
+
```
Comment (a-rosa) [inline prac_06/languages.py:7]: If you use a function and call another variable outside the function, that variable is a global constant, so python, ruby and visual_basic should be in capital letters. It is not recommended to use global variables.

**Pull request closed without merging**